data/human/free worlds 4 epilogue.txt

```diff
@@ -32,7 +32,7 @@ mission "FW Epilogue: Ijs and Katya"
 					goto excitement
 				`	"It's kind of hard to adjust to civilian life."`
 					goto adjust
-				`	"I want to explore the rest of the galaxy, but apparently that part of the game hasn't been implemented yet."`
+				`	"Is this it? Is the game over now that I completed the free worlds campaign?"`
```

Comment:
```suggestion
				`	"I want to explore the rest of the galaxy, but I don't know what's been implemented."`
```
I'm just going to try suggesting something.

Reply:
@EjoThims, this suggestion does convey "I am excited for even more ES" without the risk of confusing the player. Is this a sufficiently obvious 4th wall break? If not, how about "I want to explore the rest of the galaxy, but I don't know which parts of the game are already implemented"?

Reply:
@Zireael07 How about this version?

Reply:
This is significantly better than what you rolled out, especially in regards to at all actually matching up to Katya's response, but it's still drastically changing the meaning of the question itself, this time from an open ended but valid assumption to a presentation of ignorance from the player. It also loses the direct implication of continuing implementation from dropping that "yet." These together are the reason I didn't directly support it already as I did other continuing suggestions that I did feel were better.

A confused player is now directly being told that there's already more beyond human space in Katya's answer, though, and I still honestly don't think either this one or the one you committed to the PR actually addresses that issue any better, anyway.

Reply:
I think it's slightly better than the previous one, and I agree mostly with @EjoThims

Reply:
> A confused player is now directly being told that there's already more beyond human space in Katya's answer

Not if the player selects option 1 "I miss the excitement..." or option 2 "It's kind of hard..."

Reply:
> Not if the player selects option 1 "I miss the excitement..." or option 2 "It's kind of hard..."

Because neither of those are addressing the state of the game with a fourth wall break.

They shouldn't be given fourth wall breaking information about the game itself when they've chosen to avoid that.